LegacyAutoloader.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class LegacyAutoloader
+{
+    public function __construct()
+    {
+        spl_autoload_register(array($this, 'load_class'));
+    }
+
+    public static function register()
+    {
+        new LegacyAutoloader();
+    }
+
+    public function load_class($className)
+    {
+        if (strpos($className, 'Matomo\\') === 0) {
+            $newName = 'Piwik' . substr($className, 6);
+            if (class_exists($newName) && !class_exists($className, false)) {
+                @class_alias($newName, $className);
```

Comment:
was added in 5.3.0 so that should be fine https://www.php.net/manual/en/function.class-alias.php